crates/ruff_linter/resources/test/fixtures/flake8_bandit/S608.py

```diff
@@ -1,4 +1,6 @@
 # single-line failures
+
+var, table, val, var2, x = "", "", "", "", ""
 query1 = "SELECT %s FROM table" % (var,) # bad
```

Comment:
Let's remove this variable definitions to keep the snapshot changes to a minimum so that it's easy to review the actual changes.